src/content/core.json

```
@@ -1,190 +1,201 @@
+    "text": "Gone are the days of building for a specific standard only to have the landscape shift beneath you. Build your apps with the Safe{Core} future-proof approach standard.",
+    "component": "Wallet/ParallaxAppStore"
   },
   {
-    "caption": "Safe ecosystem builders",
-    "title": "See what web3’s best builders have to say",
+    "variant": "image-text",
+    "caption": "200+ projects",
+    "title": "<em>Safe</em> Smart Accounts power over <b>200 projects across 15+ networks</b>",
+    "buttons": [
+      {
+        "text": "Explore ecosystem",
+        "href": "https://safe.global/ecosystem",
+        "variant": "button"
+      },
+      {
+        "text": "Case studies",
+        "href": "https://safe.global/blog?category=Insights",
+        "variant": "link"
+      }
+    ],
     "items": [
       {
-        "image": {
-          "src": "/images/Core/tweets/onchainden.png",
-          "alt": "Tweet from OnchainDen"
-        },
+        "title": "Payments",
         "link": {
-          "href": "https://twitter.com/OnChainDen/status/1615363435059118080"
+          "href": "https://safe.global/blog/safe-case-2-digital-ownership-in-your-pocket"
         }
       },
       {
-        "image": {
-          "src": "/images/Core/tweets/tribes_xyz.png",
-          "alt": "Tweet from Tribes"
-        },
+        "title": "DAOs"
+      },
+      {
+        "title": "Restaking",
         "link": {
-          "href": "https://twitter.com/tribes_xyz/status/1617555624962969603"
+          "href": "https://safe.global/blog/the-safe-case-staking-and-restaking"
         }
       },
       {
-        "image": {
-          "src": "/images/Core/tweets/tarungupta.png",
-          "alt": "Tweet from tarungupta.eth"
-        },
+        "title": "AI",
         "link": {
-          "href": "https://twitter.com/tarungupta1475/status/1567088924224557062"
+          "href": "https://safe.global/blog/the-safe-case-ai-smart-accounts-crypto"
         }
       },
       {
-        "image": {
-          "src": "/images/Core/tweets/aaveaave.png",
-          "alt": "Tweet from Aave"
-        },
+        "title": "Social & ID",
         "link": {
-          "href": "https://twitter.com/AaveAave/status/1278718273933246464"
+          "href": "https://safe.global/blog/safe-case-decentralized-social-id"
         }
       },
       {
-        "image": {
-          "src": "/images/Core/tweets/ittaisvidler.png",
-          "alt": "Tweet from ittai.eth"
-        },
+        "title": "Institution",
         "link": {
-          "href": "https://twitter.com/IttaiSvidler/status/1592541969137676288"
+          "href": "https://safe.global/blog/the-safe-case-infrastructure-for-institutional-crypto-self-custody"
         }
       }
     ],
-    "component": "Core/Tweets"
+    "component": "Core/ParallaxCaseStudies"
   },
   {
-    "title": "Hacking on<br />Safe <u>CORE</u>?",
-    "caption": "Hackathon",
-    "buttons": [
+    "title": "Get started with Safe <u>CORE</u>",
```

Should this be `... Safe{Core}"`?

```suggestion
    "title": "Get Started with Safe <u>CORE</u>",
```